src/main/java/com/buglife/sdk/Client.java

```
@@ -192,24 +188,31 @@ private void onScreenshotTaken(Bitmap bitmap) {
             return;
         }
 
-        Attachment screenshotAttachment = getScreenshotBuilder().build(bitmap);
-        showAlertDialog(screenshotAttachment);
+        try {
+            // Save bitmap to a temp file
+            String filename = "screenshot_" + System.currentTimeMillis() + ".png";
+            File file = new File(mAppContext.getCacheDir(), filename);
+            bitmap.compress(Bitmap.CompressFormat.PNG, 100, new FileOutputStream(file));
+            onScreenshotTaken(file);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Exceptions shouldn't be swallowed; When an exception occurs, we should either

1. Surface it at the UI layer (if it makes sense)
2. Re-throw the exception (and mark the enclosing method as `throws`)
3. Have a "backup" code path

I still need to do a bit more review before I can know what the "right" option here is.